pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "pytest_filedata"
+repository = "git@github.com:20c/pytest-filedata.git"
+version = "0.4.0"
+description = "easily load data from files"
+authors = [ "20C <code@20c.com>",]
+license = "LICENSE.txt"
+classifiers = [
+  "Topic :: Software Development", 
+  "License :: OSI Approved :: Apache Software License", 
+  "Programming Language :: Python :: 3.6", 
+  "Programming Language :: Python :: 3.7", 
+  "Programming Language :: Python :: 3.8", 
+  "Programming Language :: Python :: 3.9",
```

Comment:
The classifiers, and everything else should be copied from setup.py as well.

Reply:
I have added the remaining classifiers. What do you mean by 'everything else'? I am a little confused because a lot of the stuff in setup.py is dynamic / is a variable / imported from a different file, so this would not work in pyproject.toml

Reply:
That might be it, but it should include everything that is defined in setup.py. I'll look later

Reply:
ok thanks, curious to see how this is translated